namedtensor/test_core.py

```diff
@@ -33,6 +33,45 @@ def test_stack_basic(data, x):
     assert not (x.shape.keys() & s)
 
 
+@given(named_tensor())
+def test_deepcopy(x):
+    import copy
+    x2 = copy.deepcopy(x)
+    assert id(x2.values) != id(x.values)
+    assert torch.equal(x2.values, x.values)
+    assert id(x2._schema) != id(x._schema)
+    assert x2._schema._names == x._schema._names
+    assert x2._schema._masked == x._schema._masked
+
+
+@given(data(), named_tensor())
+def test_unique(data, x):
+    s = data.draw(dim(x))
+    nu = data.draw(name(x))
```

Comment:
There should be a `names` function where you can draw 2 (max_size=2) that will guarantee they are unique.